bioptim/limits/phase_transition.py

```diff
@@ -57,61 +56,29 @@ def __init__(
         max_bound: float = 0,
         **params: Any,
     ):
-        """
-        Parameters
-        ----------
-        phase_pre_idx: int
-            The index of the phase right before the transition
-        params:
-            Generic parameters for options
-        """
 
         if not isinstance(transition, PhaseTransitionFcn):
             custom_function = transition
             transition = PhaseTransitionFcn.CUSTOM
-        super(Constraint, self).__init__(penalty=transition, custom_function=custom_function, **params)
-
-        self.min_bound = min_bound
-        self.max_bound = max_bound
-        self.bounds = Bounds(interpolation=InterpolationType.CONSTANT)
+        super(PhaseTransition, self).__init__(
+            min_bound=min_bound,
+            max_bound=max_bound,
+            weight=weight,
+            custom_function=custom_function,
+            first_node=Node.END,
+            second_node=Node.START,
+            **params,
+        )
 
-        self.weight = weight
-        self.quadratic = True
+        self.phase_second_idx = None
```

Comment:
Do not remove information from super if not needed

Reply:
done by you